packages/cli/src/commands/ui/create/react.spec.ts

```diff
@@ -191,18 +216,23 @@ describe('ui:create:react', () => {
 
   test
     .command(['ui:create:react', 'myapp'])
-    .it('should start a process to create the project', () => {
-      expect(mockedNpxInPty).nthCalledWith(
-        1,
-        [
-          'create-react-app@1.0.0',
-          'myapp',
-          '--template',
-          '@coveo/cra-template@1.0.0',
-        ],
-        expect.objectContaining({})
-      );
-    });
+    .it(
+      'should start a process and a transformer to create the project',
+      () => {
+        expect(mockedSpawn).nthCalledWith(
+          1,
+          expect.stringContaining('npx'),
+          [
+            'create-react-app@1.0.0',
+            'myapp',
+            '--template',
+            '@coveo/cra-template@1.0.0',
+          ],
+          expect.objectContaining({})
```

Comment:
Nitpick, but I think this could probably be transformed with `expect.any(Object)`

And is it really an empty object that we are receiving there ?

Reply:
✔️ I redid the test proper